CHANGELOG.md

```diff
@@ -24,7 +24,8 @@ recommendations of [keepachangelog.com](http://keepachangelog.com/).
 
 ### Fixed
 
-- None
+- [#594](https://github.com/paper-trail-gem/paper_trail/issues/594) -
+  In order to properly reify a version of a model using STI, item_type can refer to the class instead of base_class. When making this change, the five reifiers in the gem [paper_trail-association_tracking] that refer to base_class can also be updated, which is covered in [PT-AT PR #5](https://github.com/westonganger/paper_trail-association_tracking/pull/5). With these changes, the previously failing example in person_spec.rb passes.
```

Comment:
- Can you boil this down a bit, maybe into a single sentence?
- Should we link to #594 or this PR (#1108) or both?

Reply:
Perhaps linking to this [PR](https://github.com/paper-trail-gem/paper_trail/pull/1108) is best, and I'll make reference to [#594](https://github.com/paper-trail-gem/paper_trail/issues/594) at the top.